app/controllers/application_controller.rb

```diff
@@ -59,8 +60,11 @@ def user_not_authorized
     redirect_to(request.referer || root_path, status: :forbidden)
   end
 
+  def set_service_guide_url
+    @service_guide_url = "https://guide.manage-vaccinations-in-schools.nhs.uk"
+  end
+
   def set_privacy_policy_url
-    @privacy_policy_url =
-      current_user&.selected_organisation&.privacy_policy_url
+    @privacy_policy_url = false
```

Comment:
```suggestion
    @privacy_policy_url = nil
```

Not major, but I think this would be more typical than setting it to `false`.

Reply:
Fixed (and for `@service_guide_url` in the other controller)